### PR TITLE
[10.0][MIG] web_menu_navbar_needaction

### PR DESCRIPTION
**Pull request by hbrunn**
actual migration happens in https://github.com/OCA/web/commit/454d2dcc73a5c2e38012dbf3505baa4aed600074

### CODE REVIEW

**Comment by hbrunn** [comment]
test failures are unrelated, flake8 is fixed in https://github.com/OCA/web/pull/1029

**Review by thomaspaulb — APPROVED** [review]
This looks very neatly ported, I can't find any mistakes. Plus its counterpart in 8.0 worked really well, and there are not many differences between the versions.

File: web_menu_navbar_needaction/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+To configure the update frequency, set the configuration parameter `web_menu_navbar_needaction.refresh_timeout` to the amount of milliseconds after which the counters should be updated. Don't do this too often, as this will cause a lot of queries to the database. The default it 600000, which is equivalent to 10 minutes.
```
Comment (pedrobaeza) [inline web_menu_navbar_needaction/CONFIGURE.rst]: This should be on `readme` subdirectory

**Comment by hbrunn** [comment]
@pedrobaeza thanks, since the code sprint I actually understand what the readme generator does ;-) and done.

**Pull request merged**